### ISSUE TITLE
Mike Dickson Code for "M3 Ruth AlphaCut HUD v1.0.lsl"

### ISSUE THREAD

**Issue by SundanceHaiku** [Outworldz/Ruth#18]
Mike, your revised version of the Alpha HUD script is pretty much working in the Second Life environment.  Clicking directly on the body works fine.  However, if you click on one of the buttons, the HUD goes black.  I'm a attaching an image of what it looks like if you click on the "Chest" button.  You'll see that the alpha correctly appears on the avatar, but the HUD is obscured by black.  You'll also notice that in addition to "Chest," the button (the one I clicked),  "Butt" and "Load Alpha" buttons are also highlighted in green.  Clicking the other buttons produce similar results.  My guess is that there is an easy fix since the clicking on the body parts works fine.

![mikedixonscript](https://user-images.githubusercontent.com/34669224/34916516-bbac6748-f8f6-11e7-9a1d-8d9408470313.jpg)

**Comment by mdickson** [comment]
I'll have a look and bring what I have into SL. I have an account there.  It's probably an assumption about linkset naming but I'm sure we can work it out.

**Comment by SundanceHaiku** [comment]
Thanks Mike.  Feel free to use the Ruth Sandbox in SL.  The HUD is already
assembled, and you can just replace the old script with the new version.
Here's the SURL: http://maps.secondlife.com/secondlife/Gamma/21/76/1401

**Comment by SundanceHaiku** [comment]
Mike, sorry.  Chimera had forgotten to re-set the permissions.  It's been
corrected. The RC1 package uses Shin's original code.  Try putting your
code in and you'll see the problem.

**Comment by mdickson** [comment]
Ok this is an issue with the reset code. I got lazy there and didn't specifically use the link definitions to reset parts.  Since your copy is linked slightly differently from my own it messes up the backboard texture on a reset.  I did notice the SL version is much slower than what I see in IW. I need to check to make sure we're using the Fast versions of prim params functions as they'll add energy and delay which probably isn't added in InWorldz.
